Kluctl render --offline-kubernetes does not work with CRDs

### Operating system

Pop!_OS 22.04 (a flavor of Ubuntu)

### Kluctl Version

2.15.0

### Kubernetes Version

v1.24.4+rke2r1

### kubectl Version

v1.25.2

### Bug description

Cannot use `--offline-kubernetes` if the resources include CRDs.

### Steps to reproduce

Example `helm-chart.yaml` that will cause this error

```yaml
helmChart:
  repo: https://bitnami-labs.github.io/sealed-secrets
  chartName: sealed-secrets
  chartVersion: 2.6.9
  releaseName: sealed-secrets
  namespace: sealed-secrets
  output: chart-rendered.yaml
```

### Relevant log output

```shell
# kluctl render -t smiths.haus --offline-kubernetes --print-all --debug
Checking for new kluctl version
enter LoadKluctlProject
Loading kluctl project
Loading targets
Done loading targets
leave LoadKluctlProject
Rendering templates
Rendering Helm Charts
Building kustomize objects
Postprocessing objects
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x1be9ab4]

goroutine 5361 [running]:
github.com/kluctl/kluctl/v2/pkg/deployment.(*DeploymentItem).postprocessCRDs(0xc0010c2000)
	/home/runner/work/kluctl/kluctl/pkg/deployment/deployment_item.go:449 +0x174
github.com/kluctl/kluctl/v2/pkg/deployment.(*DeploymentCollection).buildKustomizeObjects.func3()
	/home/runner/work/kluctl/kluctl/pkg/deployment/deployment_collection.go:213 +0x33
created by github.com/kluctl/kluctl/v2/pkg/deployment.(*DeploymentCollection).buildKustomizeObjects
	/home/runner/work/kluctl/kluctl/pkg/deployment/deployment_collection.go:212 +0x374
```
```

--- a/pkg/deployment/deployment_item.go
+++ b/pkg/deployment/deployment_item.go
@@ -439,6 +439,9 @@ func (di *DeploymentItem) postprocessCRDs() error {
 	if di.dir == nil {
 		return nil
 	}
+	if di.ctx.K == nil {
+		return nil
+	}
 
 	for _, o := range di.Objects {
 		gvk := o.GetK8sGVK()

